src/util/mod.rs

```diff
@@ -73,3 +52,6 @@ pub mod address {
 
 #[deprecated(since = "0.30.0", note = "Please use crate::Error")]
 pub use crate::error::Error;
+
+#[deprecated(since = "0.30.0", note = "Please use crate::util::uint::BitArray")]
```

Comment:
In 8cde671cdebc71bba555f27acaffd0fd49decb32:

Maybe it's not for this PR, but I think we should drop the `BitArray` trait -- it is used to generalize over `Uint128` and `Uint256` which is going away. (We can leave this re-export in place for a version or two but we should tell users to just drop it, or switch to the `num` crate, or file an issue with us to say that they have some critical need for this.)

Reply:
Glad you say that, I was working on `Uint256` yesterday and wondering the utility of `BitArray`. Perhaps leave it like this for this PR and in the `Uint256` (Target/Difficulty) work I'll remove `BitArray`.

Reply:
We should go through a deprecation cycle but I really doubt there are external users. This trait is basically a grab-bag of the `num-traits` traits that I wanted (but didn't want a dependency on `num-traits`) and is nearly useless for generic code because no other popular 3rd parties implement it.

Reply:
I see you dropped the re-export. I'm fine with that, but if anyone else wants to keep it, I"ll defer to them.